src/api/index.js

```diff
@@ -6,9 +6,15 @@ export const refresh = () => logined.post('user/refresh');
 
 export const checkNickname = (nickName) => logined.get(`/user/nickname-check?nickname=${nickName}`);
 
-export const checkAccount = (account) => logined.get(`user/account-check?account=${account}-`);
+export const checkAccount = (account) => logined.get(`user/account-check?account=${account}`);
 
 export const checkEmail = (email) => logined.get(`user/email-check?email=${email}`);
 
+export const changeNickName = (nickName) => logined.post('user/nickname', nickName);
+
+export const deleteUser = () => logined.patch('user/delete');

```

Comment:
허... delete 요청인데 `patch` 메소드를 쓰나요? 백엔드 분들에게 얘기해봐야 할거 같은데...

```suggestion
export const deleteUser = () => logined.delete('user');
```

Reply:
![image](https://user-images.githubusercontent.com/50792467/148063421-b7acf705-507e-430b-a241-4608d3a01e10.png)

Reply:
물어보겠습니다;;

Reply:
회원 탈퇴기는 하지만 로그 생성 및 문제생성시 복구 등 여러가지를 위해
현재 db에서 회원정보를 삭제하지는 않습니다 회원정보에 해당 정보는 삭제됐다는 칼럼을 만듭니다.
예를 들어 is_deleted 같은 해당 필드를 수정함으로써 삭제를 나타내기에
PATCH 로 method 타입을 설정했습니다
라고 답변 받았습니다. 서버쪽에서 정하게 된거라 아마 이런식으로 갈 것 같습니다